### DISCUSSION TITLE
Follow tasks along a route

### DISCUSSION THREAD

**Discussion by oliverlorenz** [streetcomplete/StreetComplete#4363] [Ideas]
I recently discovered the app for myself and I love it! Whenever I go on a walk I try to take as many tasks with me as possible.

Unfortunately, it always shows me a lot of tasks that are not on my route. I would love a feature where I can set a route (or destination) and then primarily complete tasks along the way. 

Or alternatively: set a destination and generate a route that I can follow. Even if it's not the fastest/most efficient way.

Any thoughts in this direction already?

**Comment by mnalis** [comment]
related: https://github.com/streetcomplete/StreetComplete/discussions/3898, https://github.com/streetcomplete/StreetComplete/discussions/4288
